core/src/main/java/io/dekorate/utils/Labels.java

```diff
@@ -25,7 +25,7 @@
 
 public class Labels {
 
-  private static final String APP = "app";
```

Comment:
Why do we make it public?

Reply:
Because it is accessed from `ComponentHandler`

Reply:
I mean I know that you need to access the label from ComponentHanlder, but why don't you use the create labels method?

Reply:
Perhaps the best thing to do here would be to extract:

```
    Map<String,String> result =  new HashMap<String, String >() {{
        put(GROUP, config.getGroup());
        put(APP, config.getName());
        put(VERSION, config.getVersion());
      }};
```

into a separate method?

Otherwise calling `createLabels` here will look weird

Reply:
I don't follow. What's weird about calling createLabels?

Reply:
First of all `ComponentConfig` doesn't extend `BaseConfig`. 
Second, what we want to do here is not to populate the labels of `ComponentConfig`, but instead just add a new label to the `ComponentBuilder`.

Does that make sense? Am I missing something here?